payments-core/src/test/java/com/stripe/android/paymentsheet/paymentdatacollection/TransformToPaymentMethodCreateParamMapTest.kt

```diff
@@ -56,7 +56,7 @@ class TransformToPaymentMethodCreateParamMapTest {
                             "joe@gmail.com",
                             true
                         ),
-                        SectionFieldSpec.Country.identifier to FormFieldEntry(
+                        SectionFieldSpec.Country().identifier to FormFieldEntry(
```

Comment:
please replace this with `IdentifierSpec("country")`

Reply:
oops, sorry if I missed commenting on this